src/applications/gi/selectors/vetTecCalculator.js

```diff
@@ -13,29 +14,53 @@ export const getCalculatedBenefits = createSelector(
   getConstants,
   getInstitution,
   (inputs, constants, institution) => {
-    const inputsNull =
-      inputs.vetTecTuitionFees === null && inputs.vetTecScholarships === null;
+    const vetTecTuitionInputNull =
+      inputs.vetTecTuitionFees === null || inputs.vetTecTuitionFees === 0;
```

Comment:
You are removing the check `&& inputs.vetTecScholarships === null` from this, why?